docker/docker-compose-dcm4che.yml

```diff
@@ -0,0 +1,78 @@
+version: '3.5'
+
+services:
+
+  ldap:
+    image: dcm4che/slapd-dcm4chee:2.4.44-15.0
+    logging:
+      driver: json-file
+      options:
+        max-size: "10m"
+    ports:
+      - "389:389"
+    env_file: ./dcm4che/docker-compose-dcm4che.env
+    volumes:
+      - ./dcm4che/etc/localtime:/etc/localtime:ro
+      - ./dcm4che/etc/timezone:/etc/timezone:ro
+      - ./dcm4che/dcm4che-arc/ldap:/var/lib/ldap
+      - ./dcm4che/dcm4che-arc/slapd.d:/etc/ldap/slapd.d
+    networks:
+      - dcm4che_default
+  db:
+    image: dcm4che/postgres-dcm4chee:11.1-15
+    logging:
+      driver: json-file
+      options:
+        max-size: "10m"
+    ports:
+      - "5432:5432"
+    env_file: ./dcm4che/docker-compose-dcm4che.env
+    volumes:
+      - ./dcm4che/etc/localtime:/etc/localtime:ro
+      - ./dcm4che/etc/timezone:/etc/timezone:ro
+      - ./dcm4che/dcm4che-arc/db:/var/lib/postgresql/data
+    networks:
+      - dcm4che_default
+  arc:
+    image: dcm4che/dcm4chee-arc-psql:5.15.0
+    logging:
+      driver: json-file
+      options:
+        max-size: "10m"
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+      - "9990:9990"
+      - "11112:11112"
+      - "2575:2575"
+    env_file: ./dcm4che/docker-compose-dcm4che.env
+    environment:
+      WILDFLY_CHOWN: /opt/wildfly/standalone /storage
+      WILDFLY_WAIT_FOR: ldap:389 db:5432
+    depends_on:
+      - ldap
+      - db
+    volumes:
+      - ./dcm4che/etc/localtime:/etc/localtime:ro
+      - ./dcm4che/etc/timezone:/etc/timezone:ro
```

Comment:
is it possible to do a dir to dir map here instead of each one individually?
(I have very, very little experience with docker, so I may be way off with what's even happening here)

Reply:
Not sure whether the container will need more things within /etc folder besides localtime and timezone.. I'll check